test/unit/migration/migrator.test.ts

```diff
@@ -98,4 +98,127 @@ describe('MIGRATION: migrator', () => {
       expect(result).to.deep.equal([]);
     });
   });
+
+  describe('getJavaScriptMigrationNames', async () => {
+    it('should return the list of valid migrations under the directory.', async () => {
+      const migrationPath = await mkdtemp();
+
+      // Populate migration files to a directory.
+      await Promise.all([
+        write(path.join(migrationPath, '0001_mgr.js'), ''),
```

Comment:
Just to make it look a little aesthetically pleasing can we use real migration names here instead of `mgr.js`